lib/ae_mdw/node/db.ex

```diff
@@ -258,6 +258,9 @@ defmodule AeMdw.Node.Db do
     end
   end
 
+  @spec get_block(Blocks.block_hash()) :: key_block() | micro_block()
+  def get_block(block_hash), do: :aec_db.get_block(block_hash)
```

Comment:
:pray: for a motivation to be praised

Reply:
What you think about making public a function that calls `block_hash |> get_block() |> :aec_blocks.time_in_msecs()`?

Reply:
Besides encapsulating the Node it could simplify the tests as well.

Reply:
Good call. Done